morphir/runtime/src/org/finos/morphir/runtime/TestSummary.scala

```diff
@@ -9,7 +9,8 @@ private[runtime] case class CoverageInfo(
 case class TestSummary(
     report: String,
     countsByModule: Map[(PackageName, ModuleName), TestResultCounts],
-    private val coverage: CoverageInfo
+    private val coverage: CoverageInfo,
+    private val individualTestResults: Map[(PackageName, ModuleName), Map[String, String]]
```

Comment:
I think this should just be a map from FQName to some type which gives the test result - probably more than string.  SingleTestResult, or something?